lib/src/util.dart

```diff
@@ -15,4 +18,24 @@ class Util {
         ? truncatedDivision.toString()
         : '$truncatedDivision.$paddingZeros$remainder';
   }
+
+  /// Convert a proto [Date] instance to Dart DateTime.
+  static DateTime convertProtoDateToDateTime(
+          InsertionOrder_Budget_BudgetSegment_DateRange_Date date) =>
+      DateTime(date.year, date.month, date.day);
+
+  /// Convert a string date(yyyy/mm/dd) to Dart DateTime.
+  static DateTime convertStringDateToDateTime(String date) {
+    return DateFormat('yyyy/MM/dd').parse(date);
+  }
+
+  /// Return `true` if [target] is before [start] and after [end],
+  /// false otherwise.
+  static bool isBetweenDates(DateTime target, DateTime start, DateTime end) =>
+      (target.isAfter(start) || target.isAtSameMomentAs(start)) &&
+      (target.isBefore(end) || target.isAtSameMomentAs(end));
+
+  /// Add two string values together.
+  static String addStringRevenue(String revenueA, String revenueB) =>
+      (double.parse(revenueA) + double.parse(revenueB)).toString();
```

Comment:
can you use an int64 here instead? typically we avoid using floats to represent money

Reply:
I think the revenue are in decimals, is double our only option then?